### PR TITLE
Several fixes for docker

### PR DESCRIPTION
**Pull request by obourdon**

### CODE REVIEW

File: iface/interface.go [hunk 1]
```diff
@@ -31,39 +31,61 @@ import (
...
+}
 
 // GetMetricTypes returns list of available metric types
 // It returns error in case retrieval was not successful
-func (iface *ifacePlugin) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricType, error) {
+func (iface *ifacePlugin) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, error) {
 	metricTypes := []plugin.MetricType{}
 
-	if err := getStats(iface.stats); err != nil {
+	if err := getStats(cfg.Table(), iface.stats); err != nil {
```
Comment (marcin-krolik) [inline iface/interface.go]: What do you think about using `config.GetConfigItem(cfg, "proc_path")` before calling `getStats` and pass proper path to it?

**Comment by marcin-krolik** [comment]
@obourdon apart from above comment LGTM

File: README.md [hunk 2]
```diff
@@ -49,11 +49,18 @@ This builds the plugin in `/build/rootfs/`
 `export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/build`
 * Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-interface/blob/master/README.md#examples).
 
+Configuration parameters:
+- `proc_path` path to 'dev/net' file (helpful for running plugin in Docker container)
```
Comment (elemoine) [inline README.md]: Really `proc_path` is the path to `1/dev/net`, no?

File: README.md [hunk 3]
```diff
@@ -49,11 +49,18 @@ This builds the plugin in `/build/rootfs/`
...
+
 ## Documentation
 
 ### Collected Metrics
 List of collected metrics is described in [METRICS.md](https://github.com/intelsdi-x/snap-plugin-collector-interface/blob/master/METRICS.md).
 
+Plugin has ability to read metrics from `dev/net` file.
```
Comment (elemoine) [inline README.md]: I find this sentence confusing because the plugin will always read the data from the `dev/net` file.

**Comment by elemoine** [comment]
Just a minor comment. Otherwise LGTM.

**Comment by obourdon** [comment]
Comments should be integrated in latest commit

**Comment by elemoine** [comment]
Thanks for updating your branch @obourdon. LGTM.

`<git-dictator-mode>`
You may want to squash your commits a bit :)
`</git-dictator-mode>`

**Comment by obourdon** [comment]
@intelsdi-x/snap-maintainers could you please review and merge, thanks

**Comment by obourdon** [comment]
Note that this also fixes issue #9

**Comment by obourdon** [comment]
@marcin-krolik can this be merged please ?

**Pull request merged**